pkg/products/ups/reconciler.go

```diff
@@ -25,17 +29,15 @@ import (
 	routev1 "github.com/openshift/api/route/v1"
 
 	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
 	k8serr "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/tools/record"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	defaultInstallationNamespace = "ups"
+	DefaultInstallationNamespace = "ups"
```

Comment:
Is this now being used outside this package? Also, I'm confused as to why that's the value...we never create a namespace with that name, do we?

Reply:
oh, I was using it externally for a while,  but then went back on that decision. I've reverted this change now.